src/tests/dictionaryConcepts/components/CreateConceptForm.test.jsx

```diff
@@ -110,4 +110,36 @@ describe('Test suite for CreateConceptForm', () => {
     expect(wrapper.find('.form-group.answer')).toHaveLength(1);
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('should show all the possible options when creating another kind of conept', () => {
```

Comment:
Which are the all possible options?

Reply:
These are the components to display the name, description, answers, sets and mappings rows. I'm asserting that all these components are rendered when the concept type is not specified (i.e other).

Reply:
I have adjusted the title description to reflect that